src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/domain/LæremidlerVedtaksperiodeValideringService.kt

```diff
@@ -24,6 +25,7 @@ class LæremidlerVedtaksperiodeValideringService(
         stønadsperioder: List<StønadsperiodeBeregningsgrunnlag>,
         behandlingId: BehandlingId,
     ) {
+        brukerfeilHvis(vedtaksperioder.isEmpty()) { "Kan ikke innvilge uten vedtaksperioder." }
```

Comment:
Vi burde kanskje ha en liknende feilmelding som for tilsyn barn 👼 
```suggestion
        brukerfeilHvis(vedtaksperioder.isEmpty()) { "Kan ikke innvilge når det ikke finnes noen vedtaksperioder." }
```